modular_sojourn/bottler.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/bottle_kit
-	name = "Bottling Kit"
-	desc = "A box that holds glasses fabricators, lables, caps and quarks ready for home brewing and bottling."
+	name = "bottling Kit"
```

Comment:
This is a weird change. Did the OP say why they changed the capitalisation?

Reply:
Actually I think it's just me forgetting to decapitalize Kit to kit, thanks for this oversight.